### DISCUSSION TITLE
Remove Batteryoutput from inverter ac energy

### DISCUSSION THREAD

**Discussion by drumtrich** [WillCodeForCats/solaredge-modbus-multi#695] [General]
**Describe the feature**
As visible in the Screenshot, the HA Energy Dashboard shows the batteryoutput as solar producation. This is the reason, why I have solar production at night. Also the total consumption value of my home is false, because battery output is also counted as solar production, the consumption is twice as high.

**Screenshots**
![image](https://github.com/user-attachments/assets/176b3077-a9b6-47af-928d-b91a1205e1de)
![image](https://github.com/user-attachments/assets/f2aeb271-cbe4-474d-8019-f51f57e5dd90)
![image](https://github.com/user-attachments/assets/ad5f6790-c489-40e2-84cc-464eda6f9908)

**Home Assistant (please complete the following information):**
 - Home Assistant Core Version: 2024.10.4
 - solaredge-modbus-multi Version: 3.0.1

**Additional context**
Thank you for integration. I love it ;-)

**Comment by mpredfearn** [comment]
You need to create some template sensors (https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Template-Sensors-for-Power-and-Energy) to separate the power / energy from the PV vs Battery charge / discharge.

**Comment by drumtrich** [comment]
Yes, thanks, I solved it in a similar way yesterday, and now the Energy Dashboard looks great.

I created 3 Power Template sensors: Battery Charge, Battery Discharge, and Solar Production. For Solar Production, the battery discharge is subtracted.

For these new Power sensors, I then created 3 integral sensors using the Riemann integral and added them to the Energy Dashboard.
